src/vivarium/framework/population.py

```diff
@@ -247,8 +243,8 @@ def _order_initializers(self) -> None:
 
         self._validate_no_missing_initializers(unordered_initializers)
 
-        # This is the brute force N! way because constructing a dependency graph is work
-        # and in practice this should run in about order N time due to the way dependencies are
+        # This is the brute force N^2 way because constructing a dependency graph is work.
```

Comment:
I think I was wrong the first time, it's n**2. but it doesn't matter anymore :grin: